src/Core/Components/DataGrid/FluentDataGrid.razor

```diff
@@ -83,7 +83,13 @@
             {
                 var col = _columns[colIndex];
 
-                <FluentDataGridCell GridColumn=@(colIndex+1) Class="@ColumnClass(col)" @key="@col" TGridItem="TGridItem" Item="@item">
+                string? tooltip = null;
```

Comment:
Can be simplified to `string? tooltip = col.Tooltip ? col.RawCellContent(item) : null;`

Reply:
Done!